src/main/java/me/rigamortis/seppuku/impl/module/render/HitEffects.java

```diff
@@ -0,0 +1,102 @@
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.effect.EntityLightningBolt;
+import net.minecraft.network.play.client.CPacketPlayer;
+import net.minecraft.network.play.client.CPacketUseEntity;
+import net.minecraft.util.EnumParticleTypes;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.SoundCategory;
+import net.minecraft.util.SoundEvent;
+import net.minecraft.util.math.BlockPos;
+import team.stiff.pomelo.impl.annotated.handler.annotation.Listener;
+
+/*
+ @author Proto
+ 11/21/19
+ 
+ Dont skid bro please
+ PLEASE BRO DONT SKID
+ PLEASE BRO I WORKED SO HARD
+ ok this was made by proto ok guys!!!
+ */
+
+public final class HitEffects extends Module {
+
+    public final BooleanValue lightning = new BooleanValue("Lightning", new String[]{"Lightning"}, true);
+    public final BooleanValue explosion = new BooleanValue("Explosion", new String[]{"Explosion"}, false);
+    public final BooleanValue totem = new BooleanValue("Totem", new String[]{"Totem"}, false);
+    public final BooleanValue sounds = new BooleanValue("Sounds", new String[]{"Sounds"}, true);
+
+    protected static final Minecraft mc = Minecraft.getMinecraft();
+
+    public HitEffects() {
+        super("Hit Effects", new String[]{"Hit Effects"}, "Wacky Hit Effects", "NONE", -1, ModuleType.RENDER);
+    }
+
+    @Listener
+    public void sendPacket(EventSendPacket event) {
+        if (event.getStage() == EventStageable.EventStage.PRE) {
+            if (Minecraft.getMinecraft().world != null) {
+                    if (event.getPacket() instanceof CPacketUseEntity) {
+                        CPacketUseEntity packet = (CPacketUseEntity)event.getPacket();
+                        if (packet.getAction().equals(CPacketUseEntity.Action.ATTACK)) {
+                            Entity entity = packet.getEntityFromWorld(mc.world);
+                            ResourceLocation explodelocal = new ResourceLocation("minecraft", "entity.generic.explode");
+                            SoundEvent explosionsound = new SoundEvent(explodelocal);
+                            ResourceLocation thunderlocal = new ResourceLocation("minecraft", "entity.lightning.thunder");
+                            SoundEvent thundersound = new SoundEvent(thunderlocal);
+                            ResourceLocation lightningimpactlocal = new ResourceLocation("minecraft", "entity.lightning.impact");
+                            SoundEvent lightningimpactsound = new SoundEvent(lightningimpactlocal);
+                            ResourceLocation totemlocal = new ResourceLocation("minecraft", "item.totem.use");
+                            SoundEvent totemsound = new SoundEvent(totemlocal);
+
+                            if (lightning.getBoolean()) {
+                                EntityLightningBolt lightning = new EntityLightningBolt(mc.world, entity.posX, entity.posY + 1, entity.posZ, true);
+                                mc.world.spawnEntity(lightning);
```

Comment:
This can probably be shortened to just
```
mc.world.spawnEntity(new EntityLightningBolt(mc.world, entity.posX, entity.posY + 1, entity.posZ, true));
```
since you aren't using the variable anywhere else.